repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Run.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.pipeline.datastorage.dto
+
+import java.sql.Timestamp
```

Comment:
I think we should use `Instant` or `OffsetDateTime` instead of `Timestamp` to abstract over `sql`.
I guess `Run` class will be used in Services and Controllers, which must never "know" that we are using `SQL` somewhere.

Reply:
Changed it to use Instant